src/t2c.h

```diff
@@ -0,0 +1,10 @@
+/*
+ * rv32emu is freely redistributable under the MIT License. See the file
+ * "LICENSE" for information on usage and redistribution of this file.
+ */
+
+#include "riscv_private.h"
+
+typedef intptr_t (*funcPtr_t)(riscv_t *);
```

Comment:
Enforce the consistent style as mentioned in `CONTRIBUTING.md`.
Of course, all LLVM related APIs are exceptional.